### PR TITLE
Intrasoc article 2024

### PR DESCRIPTION
**Pull request by uday-kalyan-s**
Pls review @ItsAnkan @wildcraft958 

merge conflicts resolved

images are compressed this time

### CODE REVIEW

File: npm-shrinkwrap.json
Comment (ItsAnkan) [inline npm-shrinkwrap.json:1]: why is there shrinkwrap changes ;-;

File: src/newsletter_desc.json
Comment (ItsAnkan) [inline src/newsletter_desc.json:1]: are there unresolved conflicts ?

**Review by ItsAnkan — CHANGES_REQUESTED** [review]

File: npm-shrinkwrap.json
Comment (uday-kalyan-s) [reply npm-shrinkwrap.json:1]: oh wait there are? mb i thought I accepted incoming change il fix it up

File: src/newsletter_desc.json
Comment (uday-kalyan-s) [reply src/newsletter_desc.json:1]: fixed it

**Pull request merged**